be/src/exec/vectorized/aggregate/agg_hash_map.h

```diff
@@ -91,6 +91,28 @@ static_assert(sizeof(AggDataPtr) == sizeof(size_t));
         this->hash_map.prefetch_hash(hash_values[__prefetch_index++]); \
     }
 
+#define AGG_HASH_MAP_COMMON_METHODS()                                                                              \
```

Comment:
It's unnecessary to add this macros.
we can add this to AggHashMapWithKey

Reply:
Good advise~